### DISCUSSION TITLE
Monorepo

### DISCUSSION THREAD

**Discussion by codanator** [vikejs/vike#760] [Discussion]
Hi folks,

iam new to ssr and came across this plugin...
my question is how to properly setup a monorepo using vps and pnpm having a folder structure like this:

```
services/
 - backend/
 - - src/...
 - - vite.config.ts and so on
 - - tsconfig.json
 - frontend/
 - - src/...
 - - vite.config.ts and so on
 - - tsconfig.json
- components/
- - shared components betweeen backend and frontend
 - server/
 - - src/
 - - tsconfig.json
 - - using express, prisma and trpc as the main tools
 pnpm-workspace.yaml
```

wanting to have a backend/cms and frontend/client and server code using prisma, trpc and other cool tools...

is this possible whitout mixing things that much as in your examples?

THX for directing in me in the right direction

**Comment by brillout** [comment]
How about https://vite-plugin-ssr.com/file-structure#domain-driven instead?
